browser/views.py

```diff
@@ -624,3 +624,74 @@ def attendee_create(request, course_id):
         'form': form,
     }
     return render(request, template, context)
+
+
+@staff_member_required
+def add_investigator_record(request, person_id):
+    person = get_object_or_404(Person, pk=person_id)
+    template = "browser/investigator_create.html"
+    if request.method == 'GET':
+        form = AddInvestigationForm()
+
+    elif request.method == 'POST':
+        print("POST in add_investigator")
+        form = AddInvestigationForm(request.POST,instance=person)
+        if form.is_valid():
+            investigatorobject = Investigator(subject=form.cleaned_data.get('subject'),role=form.cleaned_data.get('role')
+                              ,person_id=person_id,year=form.cleaned_data.get('year'), changed_by=request.user)
+            investigatorobject.save()
+            template = "browser/person.html"
+
+    context = {
+        'form': form,
+        'person': person,
+    }
+
+    return render(request, template, context)
+
+
+@staff_member_required
+def edit_investigator_record(request,person_id,research_id):
+
+    person = get_object_or_404(Person, pk=person_id)
+    research = get_object_or_404(Investigator, pk=research_id)
+    template = "browser/investigator_edit.html"
+    if request.method == 'GET':
+        # Retreive investigator data already stored in database and display to the user
+        form = AddInvestigationForm(initial={'subject': research.subject, 'role': research.role,
```

Comment:
why is this form called 'AddInvestigatorForm' if it is used for adding and editing?